src/hotspot/share/cds/cppVtables.cpp

```diff
@@ -66,19 +66,20 @@
 
 class CppVtableInfo {
   intptr_t _vtable_size;
-  intptr_t _cloned_vtable[1];
+  intptr_t _cloned_vtable[1]; // Pseudo flexible array member.
+  static size_t cloned_vtable_offs() { return offset_of(CppVtableInfo, _cloned_vtable); }
 public:
   static int num_slots(int vtable_size) {
     return 1 + vtable_size; // Need to add the space occupied by _vtable_size;
```

Comment:
Pre-existing: Maybe this ought to be `byte_size() / sizeof(intptr_t)` or something like that? And the
name `num_slots` seems confusing for what this is doing.  Do we actually need both `byte_size`
and `num_slots`?

Reply:
`num_slots` is unused. Removed.